airflow/utils/decorators.py

```diff
@@ -50,3 +51,33 @@ def wrapper(*args, **kwargs):
         return func(*args, **kwargs)
 
     return cast(T, wrapper)
+
```

Comment:
Moved it here because it should be here - it was not moved from 'venv' as itis now used in more places (I left an import for back-compat there).